apps/easy-messe/components/OfferMass/OfferMassCard.tsx

```diff
@@ -110,15 +122,18 @@ export default function OfferMassCart({ massInfos }: OfferMassCartProps): JSX.El
                         <Typography
                             variant="body2"
                             textAlign="center"
-                        >Prix</Typography>
+                        >
+                            Prix
+                        </Typography>
                         <Typography
                             variant="caption"
                             textAlign="center"
                             sx={{
                                 fontWeight: 'bold',
+                                maxWidth: { laptop: '60px', mobile: 'auto' },
                             }}
                         >
-                            {mass.massInfos.price !== null && `${mass.massInfos.price} frc`}
+                            {mass.massInfos.price !== null && `${mass.massInfos.price} frs CFA`}
```

Comment:
you don't need to present the number with ~frs CFA~
`formatNumber` from `react-intl` does that for you and automatically formats the number with spaces or commas, and adds the appropriate currency.

```ts
formatNumber(mass.massInfos.price, {style: 'currency', currency: 'XAF'})
```

destructure the mass object at the top of the file.

to check for `null` or `undefine` or `empty string` or `0`, you don't need to precise those, unless you need that specific check.


```suggestion
                            {mass.massInfos.price && formatNumber(mass.massInfos.price, {style: 'currency', currency: 'XAF'})}
```